Fika.Core/EssentialPatches/FikaVersionLabel_Patch.cs

```diff
@@ -31,13 +36,52 @@ internal static void PatchPostfix(string major, object __result)
                 Logger.LogInfo($"Server version: {_versionLabel}");
             }
 
-            string fikaVersion = Assembly.GetAssembly(typeof(FikaVersionLabel_Patch)).GetName().Version.ToString();
-
             Traverse preloaderUiTraverse = Traverse.Create(MonoBehaviourSingleton<PreloaderUI>.Instance);
-
             preloaderUiTraverse.Field("_alphaVersionLabel").Property("LocalizationKey").SetValue("{0}");
-            preloaderUiTraverse.Field("string_2").SetValue($"Fika {fikaVersion} |");
-            Traverse.Create(__result).Field("Major").SetValue($"FIKA BETA {fikaVersion} | {_versionLabel}");
+            if (FikaPlugin.OfficialVersion.Value)
+            {
+                _versionLabel = $"{GameVersion()} Beta version | PvE";
+                
+                preloaderUiTraverse.Field("string_2").SetValue(_versionLabel);
+                Traverse.Create(__result).Field("Major").SetValue(_versionLabel);
+            }
+            else
+            {
+                string fikaVersion = Assembly.GetAssembly(typeof(FikaVersionLabel_Patch)).GetName().Version.ToString();
+                preloaderUiTraverse.Field("string_2").SetValue($"Fika {fikaVersion} | PvE");
+                Traverse.Create(__result).Field("Major").SetValue($"FIKA BETA {fikaVersion} | {_versionLabel} | PvE");
+            }
+        }
+
+        /// <summary>
+        /// Get exe file version
+        /// </summary>
+        /// <returns>file version</returns>
+        private static string GameVersion()
```

Comment:
Is there not a different way to do this? Like getting the running assembly?

Reply:
i don't kown i that.but this is the most easy way.i just update to 0.14.8.30268,the exe file's version code also changed.